docs/docs/quickstarts/dot-net.md

```diff
@@ -0,0 +1,122 @@
+---
+title: .NET
+---
+
+This integration guide shows you how to integrate **ZITADEL** into your .NET application.
+It demonstrates how to fetch some data from the ZITADEL management API.
+
+At the end of the guide you should have an application able read the details of your organisation.
```

Comment:
```suggestion
At the end of the guide you should have an application able to read the details of your organisation.
```

Reply:
pretty sure that this is wrong in the Go Quickstart as well ;-)

Reply:
i change it in both files